src/logger/logger.cpp

```diff
@@ -83,91 +83,98 @@ void Logger::Init()
  * Public
  **********************************************************************************************************************/
 
-void Logger::LogCallback(aos::LogModule module, aos::LogLevel level, const aos::String& message)
+void Logger::LogCallback(const char* module, aos::LogLevel level, const aos::String& message)
 {
-    switch (static_cast<int>(module.GetValue())) {
+    LogModule logModule;
+    logModule.FromString(module);
+
+    switch (logModule.GetValue()) {
```

Comment:
Instead of switch we should use if else if

Compare `module` with const string. Use Aos string.